.coveragerc

```diff
@@ -19,3 +19,4 @@ exclude_lines =
     pragma: nocover
     pragma: no${PLATFORM}
     pragma: skipif_${PY_39}_py39
+    pragma: skipif_${PY_36}_py36
```

Comment:
Needed to 'invert' the steps of getting `all_tasks()`; see [this line](https://github.com/aio-libs/aiosmtpd/pull/213/files#diff-89c287d7da41aad2a025a035e9dddd498ad0ff054ec415bdeac66629ac067b41R146)

If the steps are not inverted, Python 3.7 complains loudly.